backbone/src/main/java/org/researchstack/backbone/ui/views/FixedSubmitBarLayout.java

```diff
@@ -79,8 +79,7 @@ private void onScrollChanged(ScrollView scrollView, View submitBarGuide, View su
         if (guidePosition <= yLimit) {
             ViewCompat.setTranslationY(submitBar, 0);
         } else {
-            int translationY = guidePosition - yLimit;
-            ViewCompat.setTranslationY(submitBar, translationY);
+            ViewCompat.setTranslationY(submitBar, 0);
```

Comment:
Do you have any idea what these two `guidePosition` and `yLimit` do?

Reply:
nope, that's why i didn't remove it 
although they are not used anymore now, i left it just in case, until we know for sure that everything is still working

Reply:
Deal, can you add a comment referencing the ticket here and what you just said? I don't want to touch RStack too much without covering our backs :-) Thanks.

Reply:
done